pravega/pravega
# Issue 825: Documented config.properties

**Change log description**
1. Documented config.properties.
2. Added license header to non-code files
3. Fixed a few default values (HDFS, DurableLog)

**Purpose of the change**
Improve documentation and default values.

**What the code does**
Same thing.

**How to verify it**
Proof-read added documentation. Verify build succeeds. Verify changed default values make sense.
